T_0:
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
T_1:
- 0.9999101161956787
- 0.013313216157257557
- 0.0016014974098652601
- 0.0
- 0.008083035238087177
- -0.5031347274780273
- -0.8641701936721802
- 0.0
- -0.010699115693569183
- 0.8641054034233093
- -0.5031971335411072
- 0.0
- -0.014939302578568459
- -0.2853009104728699
- 0.09296030551195145
- 1.0
T_2:
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
- 0.0
- 0.0
- 0.0
- 0.0
- 1.0
camera_1:
  apriltag_ros_continuous_node:
    max_hamming_dist: 2
    publish_tag_detections_image: true
    publish_tf: true
    queue_size: 1
    standalone_tags:
    - id: 1
      name: tag1
      size: 0.1
    tag_blur: 0.0
    tag_bundles:
    - layout:
      - id: 1
        qw: 1
        qx: 0
        qy: 0
        qz: 0
        size: 0.1
        x: 0
        y: 0
        z: 0
      name: tag
    tag_debug: 0
    tag_decimate: 1.0
    tag_family: tag36h11
    tag_refine_edges: 1
    tag_threads: 2
    transport_hint: raw
  tag_detections_image:
    compressed:
      format: jpeg
      jpeg_optimize: false
      jpeg_progressive: false
      jpeg_quality: 80
      jpeg_restart_interval: 0
      png_level: 9
    compressedDepth:
      depth_max: 10.0
      depth_quantization: 100.0
      format: png
      png_level: 9
    theora:
      keyframe_frequency: 64
      optimize_for: 1
      quality: 31
      target_bitrate: 800000
camera_2:
  apriltag_ros_continuous_node:
    max_hamming_dist: 2
    publish_tag_detections_image: true
    publish_tf: true
    queue_size: 1
    standalone_tags:
    - id: 1
      name: tag1
      size: 0.1
    tag_blur: 0.0
    tag_bundles:
    - layout:
      - id: 1
        qw: 1
        qx: 0
        qy: 0
        qz: 0
        size: 0.1
        x: 0
        y: 0
        z: 0
      name: tag
    tag_debug: 0
    tag_decimate: 1.0
    tag_family: tag36h11
    tag_refine_edges: 1
    tag_threads: 2
    transport_hint: raw
  tag_detections_image:
    compressed:
      format: jpeg
      jpeg_optimize: false
      jpeg_progressive: false
      jpeg_quality: 80
      jpeg_restart_interval: 0
      png_level: 9
    compressedDepth:
      depth_max: 10.0
      depth_quantization: 100.0
      format: png
      png_level: 9
    theora:
      keyframe_frequency: 64
      optimize_for: 1
      quality: 31
      target_bitrate: 800000
camera_3:
  apriltag_ros_continuous_node:
    max_hamming_dist: 2
    publish_tag_detections_image: true
    publish_tf: true
    queue_size: 1
    standalone_tags:
    - id: 1
      name: tag1
      size: 0.1
    tag_blur: 0.0
    tag_bundles:
    - layout:
      - id: 1
        qw: 1
        qx: 0
        qy: 0
        qz: 0
        size: 0.1
        x: 0
        y: 0
        z: 0
      name: tag
    tag_debug: 0
    tag_decimate: 1.0
    tag_family: tag36h11
    tag_refine_edges: 1
    tag_threads: 2
    transport_hint: raw
  tag_detections_image:
    compressed:
      format: jpeg
      jpeg_optimize: false
      jpeg_progressive: false
      jpeg_quality: 80
      jpeg_restart_interval: 0
      png_level: 9
    compressedDepth:
      depth_max: 10.0
      depth_quantization: 100.0
      format: png
      png_level: 9
    theora:
      keyframe_frequency: 64
      optimize_for: 1
      quality: 31
      target_bitrate: 800000
nebula_1:
  mtof_depth:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  mtof_rgb:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  nebula_ros_driver:
    boot_order: 1
    depth_enable: true
    device_numbers: 3
    exposure_enable: false
    exposure_time: 1000
    ir_enable: true
    ir_fps: 15
    log_dir: /tmp/
    mtof_ae_mode: 0
    mtof_filter_level: 1
    point_cloud_enable: true
    resolution_mode_index: 0
    rgb_enable: true
    rgbd_enable: true
    serial_number: ''
    slam_mode: 1
    stof_filter_level: 1
    stream_sdk_log_enable: true
    usb_port_number: 3-1.1
nebula_2:
  mtof_depth:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  mtof_rgb:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  nebula_ros_driver:
    boot_order: 2
    depth_enable: true
    device_numbers: 3
    exposure_enable: false
    exposure_time: 1000
    ir_enable: true
    ir_fps: 15
    log_dir: /tmp/
    mtof_ae_mode: 0
    mtof_filter_level: 1
    point_cloud_enable: true
    resolution_mode_index: 0
    rgb_enable: true
    rgbd_enable: true
    serial_number: ''
    slam_mode: 1
    stof_filter_level: 1
    stream_sdk_log_enable: true
    usb_port_number: 3-1.2
nebula_3:
  mtof_depth:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  mtof_rgb:
    image_raw:
      compressed:
        format: jpeg
        jpeg_optimize: false
        jpeg_progressive: false
        jpeg_quality: 80
        jpeg_restart_interval: 0
        png_level: 9
      compressedDepth:
        depth_max: 10.0
        depth_quantization: 100.0
        format: png
        png_level: 9
      theora:
        keyframe_frequency: 64
        optimize_for: 1
        quality: 31
        target_bitrate: 800000
  nebula_ros_driver:
    boot_order: 3
    depth_enable: true
    device_numbers: 3
    exposure_enable: false
    exposure_time: 1000
    ir_enable: true
    ir_fps: 15
    log_dir: /tmp/
    mtof_ae_mode: 0
    mtof_filter_level: 1
    point_cloud_enable: true
    resolution_mode_index: 0
    rgb_enable: true
    rgbd_enable: true
    serial_number: ''
    slam_mode: 1
    stof_filter_level: 1
    stream_sdk_log_enable: true
    usb_port_number: 3-1.4
rosdistro: 'noetic

  '
roslaunch:
  uris:
    host_firefly__35841: http://firefly:35841/
    host_firefly__39325: http://firefly:39325/
rosversion: '1.16.0

  '
run_id: 723dac58-d9f9-11ee-bc4c-ced4a954d965
